### PR TITLE
redirect / route to /home

### PR DESCRIPTION
**Pull request by rohit1576**
This is a PR to solve issue #66 , in which the / route has to be redirected to /home route.

### CODE REVIEW

File: pclubWebsite/urls.py [hunk 1]
```diff
@@ -15,8 +15,10 @@
 """
 from django.conf.urls import url, include
 from django.contrib import admin
+from django.views.generic import RedirectView
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^home/', include('home.urls')),
+    url(r'^' ,RedirectView.as_view(url='/home')),
```
Comment (sagar-kalra) [inline pclubWebsite/urls.py]: This will not work, use regular expressions properly:- r'^$'
Just make this change then you are good to go.

Comment (sagar-kalra) [inline pclubWebsite/urls.py]: This will not work, use regular expressions properly:- r'^$'
Just make this change then you are good to go.

Comment (rohit1576) [reply pclubWebsite/urls.py]: I've checked and its working fine, and even if I make the changes as you suggested, it'll work,

Comment (gurpreetsingh00885) [reply pclubWebsite/urls.py]: Its not working @rohit1576 
It keeps redirecting infinitely. Try opening the review app.

Comment (sagar-kalra) [reply pclubWebsite/urls.py]: I tried your code, its not working
So, just make the required changes. Thank You.
screenshot from 2017-10-04 16-44-06

**Comment by rohit1576** [comment]
@sagar-kalra @gurpreetsingh00885  I've rectified the error as you suggested, now it should work fine, thanks and my apologies.

File: templates/base.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <title>{% block title %}Programming club{% endblock %}</title>
+    <title>{% block title %}Programming Club{% endblock %}</title>
```
Comment (sagar-kalra) [inline templates/base.html]: Why are you making these changes?

File: templates/base.html [hunk 3]
```diff
@@ -26,7 +26,7 @@
 <nav class="navbar navbar-inverse" style="margin-bottom:0;">
   <div class="container-fluid">
     <div class="navbar-header">
-      <a class="navbar-brand" href="{% url 'home:home' %}">Programming club</a>
+      <a class="navbar-brand" href="{% url 'home:home' %}">Programming Club</a>
```
Comment (sagar-kalra) [inline templates/base.html]: Dont make these changes.

**Comment by divyam3897** [comment]
Fixed in https://github.com/pclubuiet/website/commit/ce643e1afa7c07424f020cb052211013f0c5728a Thanks

**Pull request closed without merging**